docs/SUMMARY.md

```diff
@@ -1,17 +1,18 @@
 # Table of Content
 
-* [Scenarioo Examples](examples.md)
+* [Introduction & Overview](README.md)
+  * [Scenarioo Examples](examples.md)
+  * [Downloads and Links](tutorial/downloads-and-links.md)
 
-* [Downloads and Links](setup/downloads-and-links.md)
-
-* [Setup & Usage](setup/README.md)
-  * [Quickstart with Scenarioo-JS](setup/Quickstart-JS.md)
-  * [Install Viewer Web App](setup/Scenarioo-Viewer-Web-Application-Setup.md)
-  * [Install Viewer Docker Image](setup/Scenarioo-Viewer-Docker-Image.md)
-  * [Viewer Configuration](setup/Configuration.md)
-  * [Usage of Writer Libraries](setup/How-to-use-Scenarioo-Writer-Library.md)
-  * [Publish Documentation Data](setup/Publish-Documentation-Data.md)
-  * [Migration Guide](setup/Migration-Guide.md)
+* [Getting Started](tutorial/README.md)
+  * [Install Viewer Web App](tutorial/Scenarioo-Viewer-Web-Application-Setup.md)
+  * [Write Reports from Tests](tutorial/How-to-use-Scenarioo-Writer-Library.md)
+     * [Quickstart with Scenarioo-JS](tutorial/Quickstart-JS.md)
+     * [Use Scenarioo in Java Selenium Tests](tutorial/Using-Scenarioo-In-Java-Selenium-Tests.md)
+  * [Install Viewer Docker Image](tutorial/Scenarioo-Viewer-Docker-Image.md)
+  * [Publish Reports](tutorial/Publish-Documentation-Data.md)
+  * [Viewer Configuration](tutorial/Configuration.md)
+  * [Migration Guide](tutorial/Migration-Guide.md)
```

Comment:
Migration Guide should not be under "Getting Started", as this is only interesting for people that have been using Scenarioo for a while.

Reply:
Move to changelog